generated/python/gapic-google-cloud-datastore-v1/docs/conf.py

```diff
@@ -294,17 +289,15 @@
 # How to display URL addresses: 'footnote', 'no', or 'inline'.
 #texinfo_show_urls = 'footnote'
 
-# If true, do not generate a @detailmenu in the "Top" node's menu.
+# If true, do not generate a @etailmenu in the "Top" node's menu.
```

Comment:
Thanks @geigerj. I approved your PR.

Seems like we can just hand-edit a fix here, rather than re-generating.